api/web_publication_manifest.py

```diff
@@ -111,18 +111,21 @@ def __init__(
         # Add the SpineItems as reading order items. None of them will
         # have working 'href' fields -- it's just to give the client a
         # picture of the structure of the timeline.
+        # For the href we return a URN that includes the part and sequence
```

Comment:
Minor: Might be worthwhile to remove the second sentence of the previous paragraph.